README.md

```diff
@@ -74,7 +74,7 @@ on the most common IDEs:
 
 ### Enabling
 
-By default, ConfigManager runs in `DEBUG_MODE` off. To turn on, in the `setup` routine:
+By default, ConfigManager runs in `DEBUG_MODE` off. To turn on, change debugging to true after including `ConfigManager`:
```

Comment:
I'd add to the documentation the purpose of the debug mode, wich is, the Serial interface can be needed to communicate over serial with other device